src/renderer/components/AccountTagDerivationMode.js

```diff
@@ -23,21 +23,21 @@ const CurrencyLabel: ThemedComponent<*> = styled(Text).attrs(() => ({
   flex: 0 0 auto !important;
   box-sizing: content-box;
   text-transform: uppercase;
-  margin: 0 8px;
   flex: unset;
 `;
 
 type Props = {
   account: AccountLike,
+  margin?: string,
 };
 
-export default function AccountTagDerivationMode({ account }: Props) {
+export default function AccountTagDerivationMode({ account, margin }: Props) {
   if (account.type !== "Account") return null;
 
   const tag =
     account.derivationMode !== undefined &&
     account.derivationMode !== null &&
     getTagDerivationMode(account.currency, account.derivationMode);
 
-  return tag ? <CurrencyLabel>{tag}</CurrencyLabel> : null;
+  return tag ? <CurrencyLabel margin={margin ?? "0 8px"}>{tag}</CurrencyLabel> : null;
```

Comment:
As I know, this change was made to properly render the multiline variant of the select account component.
@jelbaz-ledger could you confirm?

Reply:
Yup (if I recall correctly) that's it. I removed the hardcoded value and added the ability to accept a prop  because the 8px margin in both sides of the tag was quite ugly when used in the multiline layout.